gcp/gcp.go

```diff
@@ -147,6 +147,7 @@ func (gcp *GoogleCloudPrint) Fetch(gcpID string) ([]Job, error) {
 	responseBody, errorCode, _, err := postWithRetry(gcp.robotClient, gcp.baseURL+"fetch", form)
 	if err != nil {
 		if errorCode == 413 {
+			log.Debugf("No jobs returned by fetch (413 error)")
```

Comment:
np, done. I think GitHub allows you to squash commits on merge now which would make this cleaner :-)